### PR TITLE
COOPR-234: Added version command to CLI.

### PR DESCRIPTION
**Pull request by tyatskiv**
@albertshau The version command has been added to the COOPR CLI in the same way, as it implemented for the CDAP.

### CODE REVIEW

File: coopr-cli/src/main/java/co/cask/coopr/shell/command/VersionCommand.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  }
+
+  @Override
+  public String getPattern() {
+    return "version";
+  }
+
+  @Override
+  public String getDescription() {
+    return "Prints the version";
```
Comment (albertshau) [inline coopr-cli/src/main/java/co/cask/coopr/shell/command/VersionCommand.java]: It is more precise to describe this as the "CLI version", as it is possible to server and provisioner are running on a different version.

**Comment by albertshau** [comment]
small comment, otherwise lgtm

**Comment by tyatskiv** [comment]
@albertshau I have updated PR, please take a look.

**Comment by albertshau** [comment]
:+1:

**Pull request merged**